src/attr.rs

```diff
@@ -17,6 +17,7 @@ pub struct Attributes {
     pub commented: bool,
     /// If (a) the first space child contains a newline, (b) one of the children is a multiline
     pub multiline: bool,
+    pub multiline_self: bool,
```

Comment:
i guess this correspond to case (a) ?

Reply:
yes, as mentioned in the PR description

Reply:
Maybe we should have a more descriptive name(`multiline => is_descent_multiline`, `multiline_self` => `is_direct_children_multiline` for example) and add doc string.